npm/src/compiler/compiler.ts

```diff
@@ -25,8 +25,24 @@ type Wasm = typeof import("../../lib/node/qsc_wasm.cjs");
 export interface ICompiler {
   checkCode(code: string): Promise<VSDiagnostic[]>;
   getHir(code: string, languageFeatures?: string[]): Promise<string>;
+  run(
```

Comment:
Extreme nit: I prefer to organize these methods such that the overloads are next to each other. Would make the diff easier to read too I think